### ISSUE TITLE
Clicking 'Begin Verification' on Key Share Request does not close initial modal

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/riot-web#11800]
### Description

![2020-01-08-16-36-16](https://user-images.githubusercontent.com/1342360/71997309-8ee13200-3235-11ea-8a47-bb15ad3c7ecd.gif)

When you set up a new device, after a while it requests keys from your other devices. On Riot-web this pops up a little modal giving you options, one of them being verifying the device.

At the moment, when you click "Start verification", it pops up a verification modal without closing the initial popup.

### Steps to reproduce

- Add a new device to your account and wait for it to request keys
- Click "Start verification"
- See the verification prompt open behind the original prompt

Logs being sent: yes

### Version information

- **Platform**: web
- **Browser**: Firefox v73.0a1
- **OS**: Debian Linux
- **URL**: riot.im/develop

**Comment by turt2live** [comment]
The keyshare is actually appearing after you accept the first one. It's been known for a while, but apparently no one has filed an issue.

**Comment by jryans** [comment]
Yes, this is quite unfortunate... 😓 At the moment, I believe cross-signing's separate flow for insisting you verify devices upfront should remove most instances of this appearing.

**Comment by MacLemon** [comment]
Same problem occurs when you get a warning, that there are “unverified devices in a room”. Clicking “Show devices” gives you a list of the unknown device IDs. Clicking on *any* Verify… button in that list will also seemingly not do anything.
In reality this pops up the verification dialog *under* the current list dialog, completely invisible to the user.

This is a *regression* 1.5.7! Prior versions did layer the verification dialog on top of the list for users to see and interact.

This also greatly impacts the ability to complete the verification workflow at all. Most users will dismiss it as “not working at all”.

**Comment by TriplEight** [comment]
> Same problem occurs when you get a warning, that there are “unverified devices in a room”. 

I would x120 this problem as everyone experiences it in our team, and this is incredibly annoying UI bug.

**Comment by kittykat** [comment]
I'm going to close this issue for now as we cannot reproduce it. Please re-open if you feel that it is still an issue.
